duckduckgo/macos-browser
# FindTime URL problem

Task/Issue URL: https://app.asana.com/0/1177771139624306/1200633405158721/f

**Description**:
The browser was decoding links opened from other apps which caused failures, for example for FindTime links

**Steps to test this PR**:
1. Export, Install, make sure DDG is default browser
2. Paste this link to notes - http://example.com/?a=http%3A%2F%2Fexample.com%2F%3Fa%3Dbc
3. Click on the link and make sure address bar contains https://example.com/?a=http%3A%2F%2Fexample.com%2F%3Fa%3Dbc instead of https://example.com/?a=http://example.com/?a=bc

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
